mitodl/micromasters
# Added country_of_residence to FinancialAid model

#### What are the relevant tickets?
Fixes #1337

#### What's this PR do?
Adds `country_of_residence` to `FinancialAid` so it gets recorded in the audit table.

#### How should this be manually tested?
Before you run any migrations, checkout master and create a financial aid application if you haven't already. Then checkout this branch, run migrations, and verify that `FinancialAid.country_of_residence` is equal to `Profile.country` for that user.

Afterwards, create a new financial aid application and verify that `country_of_residence` for that field is populated with your country of residence.
